configs/runtime.yml

```diff
@@ -1,4 +1,4 @@
-use_gpu: true
+use_gpu: false
```

Comment:
revert

Reply:
Hi, thanks for reviewing, this line is changed now. Please continue your review. Thank you very much!